### PR TITLE
Add nodejs-multiselect 0.9.12

### PR DESCRIPTION
**Pull request by dLobatog**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.13
* [ ] 1.12
* [ ] 1.11

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by dLobatog** [comment]
Wrong dep version on foreman - updated so it's **>=**

File: nodejs-multiselect/nodejs-multiselect.spec [hunk 1]
```diff
@@ -0,0 +1,58 @@
+%global npm_name multiselect
+%global enable_tests 1
+
+%{?nodejs_find_provides_and_requires}
+
+Name: nodejs-%{npm_name}
+Version: 0.9.12
+Release: 1%{?dist}
+Summary: A user-friendlier drop-in replacement for the standard select with multiple attribute activated
```
Comment (domcleal) [inline nodejs-multiselect/nodejs-multiselect.spec]: Should be shorter, IIRC it's a 72 character limit

File: nodejs-multiselect/nodejs-multiselect.spec [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+License: WTFPL
```
Comment (domcleal) [inline nodejs-multiselect/nodejs-multiselect.spec]: The licence text shipped inside the package is MIT, I don't think the metadata is correct.

**Comment by dLobatog** [comment]
Updated the license (apparently npmjs was wrong) and shortened the summary, thanks for taking a look @domcleal

**Comment by domcleal** [comment]
Thanks @dLobatog, building.

**Pull request merged**